scripts/apps/vocabularies/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const MEDIA_TYPES = {
+    media: {
```

Comment:
It sounds silly having a media type called `media`. Can you rename it to `gallery`? You also need to wrap the labels in `gettext` - You can use the global `gettext` which will at least mark the translations. We will make it work in the upcoming refactoring of translations.

Reply:
I prefer to let it called media in order to minimise the data migrations.
The text is already translated on html files.